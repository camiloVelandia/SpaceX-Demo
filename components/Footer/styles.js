import styled from "styled-components";

export const Foot = styled.footer`
 padding:20px;
 border-top:2px solid white;
`;
export const Text = styled.p`
 text-align:center;
 max-width:700px;
 margin:0 auto;
`;
