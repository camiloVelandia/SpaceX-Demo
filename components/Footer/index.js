import React from 'react';
import {Foot, Text} from './styles'

const Footer = () => {
  return (
    <Foot>
      <Text>
        This project is not affiliated with the SpaceX company or any of its
        affiliates in any way. I'm just a web developer tinkering with React who
        loves space exploration and working with APIs.
      </Text>
    </Foot>
  );
};

export default Footer;