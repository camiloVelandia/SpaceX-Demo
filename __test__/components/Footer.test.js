import React from 'react'
import {mount} from 'enzyme'
import {create} from 'react-test-renderer'
import Footer from '../../components/Footer'

describe('<Footer/>',()=>{
  const footer= mount(<Footer/>)
  test('render del componente footer',()=>{
    expect(footer.length).toEqual(1)
  });
  test('content render',()=>{
    expect(footer.find('p').text()).toEqual("This project is not affiliated with the SpaceX company or any of its affiliates in any way. I'm just a web developer tinkering with React who loves space exploration and working with APIs.")
  })
})

describe('Footer snaaapshot', ()=>{
  test('Comprobar ui del footer',()=>{
    const footer= create(<Footer/>)
    expect(footer.toJSON()).toMatchSnapshot()
  })
})