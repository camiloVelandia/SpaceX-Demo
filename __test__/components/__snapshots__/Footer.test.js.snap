// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snaaapshot Comprobar ui del footer 1`] = `
<footer
  className="styles__Foot-uqm6lj-0 iYUYXz"
>
  <p
    className="styles__Text-uqm6lj-1 hrCPiu"
  >
    This project is not affiliated with the SpaceX company or any of its affiliates in any way. I'm just a web developer tinkering with React who loves space exploration and working with APIs.
  </p>
</footer>
`;
